src/Library/Card.js

```diff
@@ -0,0 +1,97 @@
+                className="iframeLarge"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+            <div className="ecommerceCard">
+                <div className="cardBadge">
+                    <h5>New</h5>
+                </div>
+                <div className="cardImg">
+                    <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRFxtc_LfP4RWXhrXJDDQyLZlEWVrlPLLhlF6LdU9UkiWlYGZfHEsSIzqU5fS7LGe6UOlHaCtpV&usqp=CAc"/>
+                </div>
+                <div className="cardBody">
+                    <p>boAt Airdopes 131 Bluetooth Headset</p>
+                <div className="cardPrice">
+                    <h4>₹5999</h4>
+                    <h5>50% off</h5>
```

Comment:
Hi Rupam, As per my understanding, h* tags are advisable only for headings. These can be replaced here, maybe a <div> tag. Let me know your thoughts.